src/app/scripts/cac/places/cac-places.js

```diff
@@ -127,7 +127,7 @@ CAC.Places.Places = (function(_, $, moment, Routing, UserPreferences) {
             if (xCoord && yCoord) {
                 var placeCoords = [yCoord, xCoord];
                 // get travel time to destination and update place card
-                Routing.planTrip(exploreLatLng, placeCoords, date, otpOptions)
+                Routing.planTrip(exploreLatLng, placeCoords, date, otpOptions, false)
```

Comment:
Having `Routing.planTrip` switch its return type based on an argument seems not ideal to me.  This is only invoked here and in `cac-control-directions.js`, right?  Couldn't we just tack `.promise()` onto the end of this line and have the function always return a Deferred?

Reply:
Given that it's convention everywhere else in this app and recommended by jQuery to always pass around non-mutable promises, I thought keeping that as the default behavior would be the least confusing thing to do. Could change that if you feel strongly otherwise.

Reply:
I see what you mean.  Though this still looks weird to me, at least it's explicit.  I don't feel strongly either way.